compair/static/modules/home/home-partial.html

```diff
@@ -20,10 +20,16 @@
 
         <h2 class="col-sm-8">Your courses</h2>
 
-        <form class="col-sm-4 search-courses text-right" role="search" ng-show="canAddCourse && (courses.length || courseFilters.search)">
+        <form class="col-sm-4 search-courses text-right" role="search" ng-show="canAddCourse && (courses.length || courseFilters.search || !courseFilters.includeSandbox)">
             <div class="form-group">
                 <input class="form-control" type="text" name="search" placeholder="Filter courses by keyword"
                        ng-model="courseFilters.search">
+                <br>
+                <label for="includeSandbox">
```

Comment:
Can you use a dropdown instead with default `All Courses` (`courseFilters.sandbox = false`) and second option `All Sandbox Courses`  (`courseFilters.sandbox = true`)